net/remote/remote-node.sh

```diff
@@ -172,10 +172,11 @@ local|tar)
     fi
 
     set -x
+    set +e
     if [[ $skipSetup != true ]]; then
       ./multinode-demo/setup.sh -t fullnode $setupArgs
     fi
-    ./multinode-demo/fullnode.sh $maybeNoLeaderRotation "$entrypointIp":~/solana "$entrypointIp:8001" > fullnode.log 2>&1 &
+    ./multinode-demo/fullnode.sh $maybeNoLeaderRotation "$entrypointIp":~/solana "$entrypointIp:8001" > fullnode.log 2>&1 & disown
```

Comment:
Can `disown` be moved to the next line?

Reply:
Actually how about moving the `disown` to immediately after the `case $nodeType in` so that we disown for all node types

Reply:
👍